src/cadenza.js

```diff
@@ -608,6 +616,26 @@ function validGeometryType(/** @type string */ value) {
   ].includes(value);
 }
 
+/** @typedef {'columns' | 'values' | 'totals'} TablePart - A part of a table to export. */
+
+const TablePart = /** @type {Record<string, TablePart>} */ {
```

Comment:
This mapping is unused in Cadenza JS, an array of the values would be sufficient.

Reply:
Should the client code be able to use those constants? (if our code does not use them directly)

I can remove them just making sure it is really intended to obliterate them.
I see for geometries we do not have defined individual constants just an array of the possible values (as you suggest here) but for MediaTypes we have constants (which we also use in our code directly). Which of the 2 approaches is best?

Reply:
The geometry types are used inside of the lib. If clients should be able to use this mapping, it needs to be exported and should be marked as @readonly and really made readonly using Object.freeze().